src/bsoncxx/types.hpp

```diff
@@ -346,10 +346,6 @@ BSONCXX_INLINE bool operator==(const b_date& lhs, const b_date& rhs) {
 ///
 /// A BSON null value.
 ///
-/// @warning
-///   This BSON type is used internally by the MongoDB server - use by clients
-///   is discouraged.
-///
```

Comment:
This is copy-pasted several other places in this file. Lets get the others while we are at it.